### DISCUSSION TITLE
Error opening connection - jserialcomm

### DISCUSSION THREAD

**Discussion by giacummo** [winder/Universal-G-Code-Sender#2511] [General]
Hello, 

I just install UGS in a clean debian linux laptop, the same version the of USG as I have in a windows notebook; but when I lunch USG for the first time, and the folowings, it say:

Error opening connection: could not connect to controller on port jserialcomm://ttyACM0:115200

I try modifing the baud rate to others values, but is the same, I change de java from open to oracle, but is the same... what can I do?

PD: I have reinstalled linux in this box, with the previous installation, an old linux flavor, it run OK, but with the new one

regards

**Comment by giacummo** [comment]
Oh!! something else, with the Arduino  IDE, it work fine.

**Reply by breiler** [reply]
Make sure that you are not connected with the Arduino IDE at the same time. Only one application may access the device at a time.

Are you sure that you have the correct user permissions to access the serial device? A possible explanation why it is working with the Arduino IDE and not UGS is if you are running the Arduino IDE as a super user? 
If that is the case, check this article:
https://support.arduino.cc/hc/en-us/articles/360016495679-Fix-port-access-on-Linux#dialout


>  I change de java from open to oracle, but is the same... 

What do you mean by this? UGS (the platform edition) comes bundled with Java so there is no need to install it separatly.

**Reply by giacummo** [reply]
Hello,

No, I dont run Arduino at the same time, I never run it, I did it just to see if the problem is the Arduino or the USG, and I dont do it as root/su

I am going to check the article later.

regards

**Reply by giacummo** [reply]
Great!!
sudo usermod -a -G dialout <username>

this solve the problem

thank you

**Comment by giacummo** [comment]
the problem was solved
